### PR TITLE
Delete-cv

### PR DESCRIPTION
**Pull request by KhesimReid**
Added an API call to remove CVs from dropbox once an applicant has cancelled their application.

### CODE REVIEW

File: src/services/applications/applicantService.ts [hunk 1]
```diff
@@ -123,6 +124,22 @@ export class ApplicantService implements ApplicantServiceInterface {
     return result;
   };
 
+  public deleteCV = async (fileName: string): Promise<string> => {
+    if (!process.env.DROPBOX_API_TOKEN) {
+      throw new Error("Failed to delete CV from Dropbox, set dropbox envs correctly");
+    }
+    if (!process.env.DROPBOX_API_TOKEN) throw new Error("Failed to delete CV from Dropbox, set dropbox envs correctly");
```
Comment (amishshah) [inline src/services/applications/applicantService.ts:131]:
```suggestion
```

Looks like it's just a duplicate of the check above.

Comment (KhesimReid) [reply src/services/applications/applicantService.ts]: Sorry about that, it was a duplicate. I just removed it.

**Comment by seanjparker** [comment]
I think this should be closed in favour of #42, it adds a new service which contains the delete method from this PR

**Comment by KhesimReid** [comment]
Closed in favour of #42.

**Pull request closed without merging**